app/form_models/agent_creneaux_search_form.rb

```diff
@@ -3,7 +3,7 @@
 class AgentCreneauxSearchForm
   include ActiveModel::Model
 
-  attr_accessor :organisation_id, :service_id, :motif_id, :agent_ids, :team_ids, :user_ids, :lieu_ids, :context
+  attr_accessor :organisation_id, :service_id, :motif_id, :team_ids, :user_ids, :lieu_ids, :context
```

Comment:
Mmm, ça me fait “unknown attribute 'agent_ids' for AgentCreneauxSearchForm” quand j’accède à la recherche de créneaux. 

C’est quoi le bug exactement? Je suis pas certain de bien comprendre. Qu’est-ce qu’il faut afficher quand on cherche à la fois des agents et des équipes?

Reply:
J'ai bricolé un truc pour que `agent_ids` renvoie une combinaison des `agent_ids` qui ont été passé en paramètre tel quel + les `agent_ids` extrait des équipes qui ont été passée en paramètre.

Reply:
Je n'aurais pas du l'enlevé  :)

Reply:
basculé en `atttr_writer` suite à notre discussion :bow: